src/templates/ProductManageTemplate.js

```diff
@@ -0,0 +1,40 @@
+import { SELECTOR } from '../constants.js';
+
+function productTemplate(name, price, quantity) {
+  return String.raw`
+		<tr>
+			<td>${name}</td>
+			<td>${price}</td>
+			<td>${quantity}</td>
+		</tr>
+	`;
+}
+
+export default state => String.raw`
+  <h3>상품 추가하기</h3>
+  <div class="product-container">
+    <input type="text" id="${SELECTOR.PRODUCT_NAME_INPUT_ID}" placeholder="상품명" />
+    <input type="number" id="${SELECTOR.PRODUCT_PRICE_INPUT_ID}" placeholder="가격" />
+    <input type="number" id="${SELECTOR.PRODUCT_QUANTITY_INPUT_ID}" placeholder="수량" />
+    <button id="${SELECTOR.PRODUCT_ADD_BUTTON_ID}">추가하기</button>
+  </div>
+  <table class="product-inventory">
+    <colgroup>
+      <col style="width: 140px" />
+      <col style="width: 100px" />
+      <col style="width: 100px" />
+    </colgroup>
+    <thead>
+      <tr>
+        <th>상품명</th>
+        <th>가격</th>
+        <th>수량</th>
+      </tr>
+    </thead>
+    <tbody id="${SELECTOR.PRODUCT_INVENTORY_CONTAINER_ID}">
+      ${state.products
+        .map(product => productTemplate(product.name, product.price, product.quantity))
+        .join('')}
```

Comment:
```suggestion
        .map(productTemplate)
        .join('')}
```
`productTemplate`도 인자들을 객체 형태로 받으면 이렇게 바꿀 수 있겠네요!